pennylane_lightning/core/src/bindings/Bindings.cpp

```diff
@@ -15,7 +15,10 @@
  * @file Bindings.cpp
  * Export C++ functions to Python using Pybind.
  */
+#ifndef _ENABLE_PLTENSOR
 #include "Bindings.hpp"
+#endif
+
```

Comment:
The initial idea was to always have the header `Bindings.hpp` included this would be the place where the base and device-specific bindings would come together to be included in this cpp file. I noticed that we have a deviation with the PLGPU_MPI, but is it possible to have this working this way for the LightningTensor?

Reply:
Thanks @AmintorDusko . I tried to keep `Bindings.hpp` here for `lightning.tensor` backend here by implementing everything following the `PLGPU_MPI` but failed. The main difference between PLGPU_MPI and LTENSOR is that PLGPU_MPI has the same feature as other `sv` backends, while `lightning.tensor` doesn't. If this is a critical issue to you, I can move LTENSOR related stuff into `BindingLTensor.cpp`. We can come back to it later if `lightning.tensor` can deliver the same features as sv backends. Let me know your thoughts.

Reply:
Removed!